tinymod/plugins/questions.py

```diff
@@ -0,0 +1,10 @@
+from hata import Client, Guild, Message, User
+
+TinyMod: Client
+GUILD: Guild
+
+@TinyMod.interactions(guild=GUILD)
+async def question(_, user: User):
+  """Recommends reading How To Ask Questions The Smart Way to a user."""
+  # TODO: explude roles?
```

Comment:
don't need role exclusion here, can just do that in server :)

Reply:
😲 Thanks for letting me know!!! Haven't had any discord bot experience since before the interactions and "you can't read all messages anymore" fiasco a couple years ago. Need to read up on HATA too 🤔 much different than what I was used to w/ discord.py.